doc/UsersGuide/source/CAPE2020.rst

```diff
@@ -0,0 +1,166 @@
+
+.. note:: 
+   
+   In order to generate all necessary configuration, data, input, and fix files to run in this method, the user first needs to run the base RT script using ``./rt.sh -n`` to set up the run directory. 
+
+To generate cold start ICs via the UFS_UTILS ``gdas_init``/``chgres_cube`` utilities on an :term:`RDHPCS` with :term:`HPSS` access (e.g., Hera or Jet), the user can run the following commands:
+
+.. code-block:: console 
+   
+   git clone https://github.com/ufs-community/UFS_UTILS.git
+   cd UFS_UTILS
+   cd fix
+   ./link_fixdirs.sh emc <platform>
+   cd ..
+   ./build_all.sh
+   cd util/gdas_init
+
+where ``<platform>`` is Hera or Jet. 
+
+Then, users will need to edit the ``config`` file to: 
+
+* Set paths for data extraction and converted files (``EXTRACT_DIR`` and ``OUTDIR``, respectively)
+* Set  ``yy/mm/dd/hh`` to desired forecast start time +6 hours
+* Set CDUMP to ``gfs``  to generate GFS ICs
+* Set ``LEVS`` to 128
+* Set ``EXTRACT_DATA`` to ``yes`` (unless data is already staged in ``EXTRACT_DIR``)
+* Set ``CRES_HIRES`` to desired resolution for coldstart files (e.g., ``C192``)
+
+In the machine-specific driver (e.g., ``driver.jet.sh``), users will need to set ``PROJECT_CODE`` to an account that the user can use for submitting batch jobs. 
+
+After configuring the utility, users can run the driver script: 
+
+.. code-block:: console
+   
+   ./driver.<platform>.sh
+
+These steps will (1) pull raw GFS data from HPSS into ``EXTRACT_DIR`` and then (2) convert the raw data to coldstart ICs (placed in ``OUTDIR/gfs.YYYYMMDD/HH/model_data/atmos/input``). The converted cold start ICs can then be used as input data for certain UFS WM regression tests (RTs) of corresponding model resolution and configuration. 
+
+To use the converted coldstart files with the UFS WM, copy the ``gfs*.nc`` and ``sfc*.nc`` files from ``OUTDIR/gfs.YYYYMMDD/HH/model_data/atmos/input`` into the ``INPUT`` directory of the UFS WM RT run directory, and adjust the model start time in the ``model_configure`` file. Note that the resolution of the converted cold start GFS data must match the ATM resolution of the UFS WM RT (e.g., use C48 GFS cold start data in the ``control_c48.v2.sfc`` RT). Then, edit the model start year, month, day, and hour in the RT ``model_configure`` file. For example: 
```

Comment:
@gspetro-NOAA this applies for changing the date of the ICs, but if users change the resolution, they will need to re-configure the test to use appropriate *.IN, input.nml, and fix files.

Reply:
I believe I've included this in the Test Configuration section (line 115-ish) already. Is that sufficient? Or do I need to restate it here...? Maybe I should just move this section down to configuration (since it applies once people have the ICs).

Reply:
maybe a brief disclaimer here that copying the ICs in in a pre-existing rundir will only work for ICs of the same resolution (and to check the Test Config. section for further info)?